From 03de0032689824422ac7fa376fb2c906ae27ecf0 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Sun, 18 Aug 2013 00:06:35 +0530
Subject: [PATCH 5/7] mero: let cs_request_handler_start() succeed

Note, that m0_fol_init() is not called in mero/setup.c, since `mero' subsystem
has not been converted to BE yet.

The following command passes now:
    m0 run-ut -t fol-ut,be-ut
---
 mero/setup.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/mero/setup.c b/mero/setup.c
index 0251e0e..35ea885 100644
--- a/mero/setup.c
+++ b/mero/setup.c
@@ -1367,8 +1367,7 @@ static int cs_request_handler_start(struct m0_reqh_context *rctx)
 #if XXX_USE_DB5
 	rc = m0_fol_init(&rctx->rc_fol, &rctx->rc_db);
 #else
-	M0_IMPOSSIBLE("XXX Not implemented");
-	rc = m0_fol_init(&rctx->rc_fol, NULL, NULL, NULL);
+	/* XXX TODO */
 #endif
 	if (rc != 0) {
 		M0_LOG(M0_ERROR, "m0_fol_init");
-- 
1.8.3.2

